src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -852,12 +891,44 @@ def update_policy_definition(policy_definition_name, rules=None, params=None,
     return policy_client.policy_definitions.create_or_update(policy_definition_name, parameters)
 
 
+def update_policy_setdefinition(policy_set_definition_name, definitions=None, params=None,
+                                display_name=None, description=None):
+    if definitions:
+        if os.path.exists(definitions):
+            definitions = get_file_json(definitions)
+        else:
+            definitions = shell_safe_json_parse(definitions)
+
+    if params:
+        if os.path.exists(params):
+            params = get_file_json(params)
+        else:
+            params = shell_safe_json_parse(params)
+
+    policy_client = _resource_policy_client_factory()
+    definition = policy_client.policy_set_definitions.get(policy_set_definition_name)
+    # pylint: disable=line-too-long,no-member
+    PolicySetDefinition = get_sdk(ResourceType.MGMT_RESOURCE_POLICY, 'PolicySetDefinition', mod='models')
+    parameters = PolicySetDefinition(
+        policy_definitions=definitions if definitions is not None else definition.policy_definitions,
+        description=description if description is not None else definition.description,
+        display_name=display_name if display_name is not None else definition.display_name,
+        parameters=params if params is not None else definition.parameters)
```

Comment:
Recommend you change "params" to "parameters" for consistency with the group deployment commands. Additionally, just like `group deployment create`, and as recommended for the policy object, I'd recommend you simply make the argument take a JSON string, and call out the `@{file}` syntax to load from a file.

Reply:
Since you are going for parity with the `policy definition create` command, this is fine.